pkg/sdk/go/certs_test.go

```diff
@@ -166,39 +175,38 @@ func TestViewCert(t *testing.T) {
 		desc   string
 		certID string
 		token  string
+		svcRes certs.Cert
+		svcErr error
 		err    errors.SDKError
-		svcerr error
-		cRes   certs.Cert
 	}{
 		{
-			desc:   "get existing cert",
+			desc:   "view existing cert",
 			certID: validID,
 			token:  token,
-			cRes:   c,
+			svcRes: cert,
+			svcErr: nil,
 			err:    nil,
-			svcerr: nil,
 		},
 		{
-			desc:   "get non-existent cert",
-			certID: "43",
+			desc:   "view non-existent cert",
+			certID: invalid,
 			token:  token,
-			cRes:   c,
+			svcRes: certs.Cert{},
+			svcErr: errors.Wrap(svcerr.ErrNotFound, repoerr.ErrNotFound),
 			err:    errors.NewSDKErrorWithStatus(errors.Wrap(apiutil.ErrValidation, svcerr.ErrNotFound), http.StatusNotFound),
-			svcerr: errors.Wrap(svcerr.ErrNotFound, repoerr.ErrNotFound),
 		},
 		{
-			desc:   "get cert with invalid token",
+			desc:   "view cert with invalid token",
```

Comment:
```suggestion
			desc:   "view cert with missing token",
```

Add another case for invalid token